client/src/utils/distance.js

```diff
@@ -9,7 +9,9 @@ export function findDistance(point1, point2) {
   const pointDistance = distance.default(from, to, options);
   return pointDistance;
 }
+/* W.8.23.20 Q: Why is this not exported? Does it not need to be? 
 
+Also can I convert this to AXIOS? */ 
```

Comment:
IMO it would be better to avoid adding another dependency like axios for just one block of logic if possible, but I will leave that up to you.